tests/Microsoft.Maui.IntegrationTests/AppleTemplateTests.cs

```diff
@@ -0,0 +1,54 @@
+
+using Microsoft.Maui.IntegrationTests.Apple;
+
+namespace Microsoft.Maui.IntegrationTests
+{
+	public class AppleTemplateTests : BaseBuildTest
+	{
+		Simulator TestSimulator = new Simulator();
+
+		[SetUp]
+		public void AppleTemplateSetup()
+		{
+			TestSimulator.Shutdown();
+			Assert.IsTrue(TestSimulator.Launch(), $"Failed to boot simulator with UDID '{TestSimulator.GetUDID()}'.");
+			TestSimulator.ShowWindow();
+		}
+
+		[OneTimeTearDown]
+		public void AppleTemplateFxtTearDown()
+		{
+			TestSimulator.Shutdown();
+		}
+
+		[Test]
+		[TestCase("maui", "Debug", "net6.0")]
+		[TestCase("maui", "Release", "net6.0")]
+		[TestCase("maui", "Debug", "net7.0")]
+		[TestCase("maui", "Release", "net7.0")]
+		[TestCase("maui-blazor", "Debug", "net6.0")]
+		[TestCase("maui-blazor", "Release", "net6.0")]
+		[TestCase("maui-blazor", "Debug", "net7.0")]
+		[TestCase("maui-blazor", "Release", "net7.0")]
+		public void RunOniOS(string id, string config, string framework)
+		{
+			if (!TestEnvironment.IsMacOS)
+				Assert.Ignore("iOS run template tests only run on macOS.");
```

Comment:
It should be possible to build on Windows, even if there's no Mac connected (there might not be any useful output, but `dotnet build` should succeed)

Reply:
The TemplateTests class contains build tests which will run on both macOS and Windows. I've filtered these run-specific tests to macOS as they attempt to boot a simulator and run Xcode tooling. I should move this check to the test or test fixture Setup though.